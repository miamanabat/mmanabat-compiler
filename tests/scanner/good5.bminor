3 % 5
2 == 889
300000925435324
-029347387
+3724883
