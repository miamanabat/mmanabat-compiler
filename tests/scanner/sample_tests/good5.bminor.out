STRING LITERAL       This is a string
STRING LITERAL       Notre 
 Dame
STRING LITERAL                 
STRING LITERAL        string with numbers 3213
STRING LITERAL       String with @@ works in the program
STRING LITERAL                 
